src/QirRuntime/lib/QIR/bridge-rt.ll

```diff
@@ -370,26 +386,46 @@ define dllexport %Callable* @__quantum__rt__callable_make_controlled(%Callable*
   ret %Callable* %.clb_cnt
 }
 
-define dllexport void @__quantum__rt__callable_memory_management(i32 %index, %Callable* %.clb, i64 %parameter) {
+; After both the compiler and runtime are in sync with https://github.com/microsoft/qsharp-language/pull/83:

```

Comment:
I think this is ready to come out with this PR, right?

Reply:
Yes.